### PR TITLE
Add a yaml file and make some modifications.

### PR DESCRIPTION
**Pull request by Summer-Field**
1. Add sc.yaml for creating storageclass without UI.
2. Modify the starter-s3.yaml to avoid interrupting kitex-contrib's normal operation.

### CODE REVIEW

File: sc.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# yaml file for creating StorageClass
+#
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+  name: test
```
Comment (Jeffwan) [inline sc.yaml]: Let's use full name `storage_class.yaml`.  Can you change `test` to `default`?

BTW, have you validate the yaml by submitting to cluster?

**Review by Jeffwan — CHANGES_REQUESTED** [review]

File: starter-s3.yaml [hunk 2]
```diff
@@ -127,20 +127,10 @@ data:
         - do-not-merge/work-in-progress
         - do-not-merge/invalid-owners-file
         orgs:
-        - kitex-contrib
+        - kitex-contrib/test-infra
```
Comment (Jeffwan) [inline starter-s3.yaml:130]: This won't work... is `kitex-contrb/test-infra` an org?  should not you use `repos`?

**Review by Jeffwan — CHANGES_REQUESTED** [review]

File: sc.yaml (hunk 1 above)
Comment (Summer-Field) [reply sc.yaml]: OK I'll change the sc.yaml to storage_class.yaml.
I've validated the yaml by submitting to cluster, it works.

File: starter-s3.yaml (hunk 2 above)
Comment (Summer-Field) [reply starter-s3.yaml:130]: My bad....

**Review by Jeffwan — APPROVED** [review]

**Pull request merged**